pkg/scheduler/scheduler.go

```diff
@@ -537,12 +537,13 @@ func (sched *Scheduler) scheduleOne() {
 		for _, pl := range plugins.PrebindPlugins() {
 			approved, err := pl.Prebind(plugins, assumedPod, scheduleResult.SuggestedHost)
 			if err != nil {
-				approved = false
```

Comment:
what happens if a plugin returns `true, err` (which shouldn't happen), this is more of defensive check I'd leave it here.

Reply:
Okay. l will turn it back